.gitignore

```diff
@@ -1 +1,2 @@
 .*cpython-310.pyc
+.devcontainer
```

Comment:
I would have preferred this commit not be included in the PR, I'll keep only the other one